scripts/pi-hole/php/savesettings.php

```diff
@@ -185,6 +195,66 @@ function readAdlists()
 	return $list;
 }
 
+function addDHCPLease($mac, $ip, $hostname) {
+	$result = array('message' => "", 'value' => false);
```

Comment:
Please don't mix `'...'` and `"..."`. This also applies to other lines in this PR.
The style of this file seems to be in favor of double-quotes.